src/sidebars/search/search-controller.js

```diff
@@ -17,8 +18,35 @@
                     selectionService.select(stock);
                 };
 
+                self.onSearchKeyUp = function(event) {
+                    if (event.keyCode === 38) {
+                        // Up
+                        changePointer(-1);
+                    } else if (event.keyCode === 40) {
+                        // Down
+                        changePointer(1);
+                    }
+                };
+
+                function changePointer(delta) {
+                    var newPointer = self.stockSelectionPointer + delta;
+
+                    self.stockSelectionPointer = Math.max(
+                        0,
+                        Math.min(
+                            newPointer,
+                            self.stocks.length - 1
+                        )
+                    );
```

Comment:
Does this mean that you can't arrow up from the top search result back into the search box? Similarly can you key down from the search box into the results? (Or do you think that is out of the scope of this?)